sherpa/__init__.py

```diff
@@ -768,7 +768,8 @@ def get_config():
     return os.path.join(os.path.dirname(__file__), filename)
 
 
-def smoke(verbosity=0, require_failure=False, fits=None, xspec=False, ds9=False):
+def smoke(verbosity=0, require_failure=False, fits=None, xspec=False,
```

Comment:
The changes in this module are purely - as far as I can see - code-formatting changes so I think they should just be removed. The PR is going to likely need a rebase to get it merged so it shouldn't be too hard to remove.

Reply:
I've reverted it, but my git-fu is not good enough to also remove this accidential change from the history unless I squach the entire PR into one commit. Do you want me to do that?